node/env0-deploy-cli.js

```diff
@@ -78,8 +78,8 @@ const getEnvironmentVariablesOptions = (environmentVariables) => {
   if (environmentVariables && environmentVariables.length > 0) {
     console.log('getting Environment Variables from options:', environmentVariables);
     environmentVariables.forEach(config => {
-      const configArray = config.split(/=(.+)/);
-      if (configArray.length === 3) {
+      const configArray = config.split('=');
```

Comment:
This is going to be problematic when the value contains a '=' for example it is a base64: 
`b21yeSBoYXkgCg==`